Brewfile

```diff
@@ -1,4 +1,4 @@
-tap 'pact-foundation/pact-ruby-standalone'
+tap 'pact-foundation/pact-ruby-standalone', 'https://github.com/pact-foundation/homebrew-pact-ruby-standalone.git'
```

Comment:
Why this change? Does this cause it to always install latest from git?

Reply:
I saw TravisCI get "stuck" waiting for someone to enter the git username. Apparently if it is `https://` it doesn't need to log in. 🤷‍♂